packages/ui/VirtualList/VirtualList.js

```diff
@@ -175,26 +345,48 @@ VirtualGridList.displaytName = 'ui:VirtualGridList';
 
 VirtualGridList.propTypes = /** @lends ui/VirtualList.VirtualGridList.prototype */ {
 	/**
-	 * Size of an item for the `VirtualGridList`.
+	 * A callback function that receives a reference to the `scrollTo` feature.
+	 *
+	 * Once received, the `scrollTo` method can be called as an imperative interface.
 	 *
-	 * * Valid value is an object that has `minWidth` and `minHeight` as properties.
+	 * The `scrollTo` function accepts the following parameters:
+	 * - {position: {x, y}} - Pixel value for x and/or y position
+	 * - {align} - Where the scroll area should be aligned. Values are:
+	 *   `'left'`, `'right'`, `'top'`, `'bottom'`,
+	 *   `'topleft'`, `'topright'`, `'bottomleft'`, and `'bottomright'`.
+	 * - {index} - Index of specific item. (`0` or positive integer)
+	 *   This option is available for only `VirtualList` kind.
```

Comment:
> for only `VirtualList`

"for `VirtualList` and `VirtualGridList`" would be better to me.

Reply:
I think with kind is better.